terraform/aws/public-cloud/main.tf

```diff
@@ -113,9 +116,9 @@ module "docker_client_certs" {
   source                = "github.com/Capgemini/tf_tls/docker/client"
   ca_cert_pem           = "${module.ca.ca_cert_pem}"
   ca_private_key_pem    = "${module.ca.ca_private_key_pem}"
-  ip_addresses_list     = "${concat(aws_instance.master.*.private_ip, aws_instance.worker.*.private_ip)}"
-  deploy_ssh_hosts      = "${concat(aws_instance.master.*.public_ip, aws_instance.worker.*.public_ip)}"
-  docker_client_count   = "${var.masters + var.workers}"
+  ip_addresses_list     = "${concat(aws_instance.edge-router.*.private_ip, concat(aws_instance.master.*.private_ip, aws_instance.worker.*.private_ip))}"
```

Comment:
Yeah, that's the quickest way I found to concat three of them masters, workers and edge-routers